tests/e2e/support/page/runtime.ts

```diff
@@ -10,7 +10,11 @@ export class RuntimePage {
   async navigateToApp({ name }: { name: string }): Promise<void> {
     const { page } = this.actor
     await page.locator('#_appSwitcherButton').click()
-    await page.locator(`a[href="#/${name}"]`).click()
+    await page
+      .locator(
+        `//ul[contains(@class, "applications-list")]//a[@href="#/${name}" or @href="/${name}"]`
```

Comment:
oCIS server now will not have `#` in the URL paths

Reply:
But will just deleting all the `#`s work for both oCIS and OC10? @fschade @phil-davis

Reply:
nope won't work in oc10

Reply:
The only way is see to get this backend agnostic is to not rely on href selectors....

Reply:
fixed